data/i18n/hi/hi.toml

```diff
@@ -35,7 +35,10 @@ other = "आप इस कुबेरनेट्स संस्करण क
 other = " प्रलेखन अब सक्रिय रूप से मेंटेन नहीं है। वर्तमान में आप जो संस्करण देख रहे हैं वह एक स्टैटिक स्नैपशॉट है। अप-टू-डेट प्रलेखन के लिए, देखें "
 
 [deprecation_file_warning]
-other = "डेप्रिकॉटेड"
+other = "डेप्रिकेटड"
```

Comment:
```suggestion
other = "डेप्रिकेटेड"
```

Reply:
Thanks for this suggestion @amolmote. [In a previous comment,](https://github.com/kubernetes/website/pull/36204#discussion_r954216012) @Garima-Negi suggested "डेप्रिकेटड". 

Garima, do you agree with this new suggestion?

Reply:
Updated. Thanks, Amol.